Library/Homebrew/shims/scm/git

```diff
@@ -92,7 +91,7 @@ case "$(lowercase "$SCM_FILE")" in
     ;;
 esac
 
-brew_version="$(quiet_safe_cd "$SCM_DIR/../../../../bin" && pwd -P)/$SCM_FILE"
+brew_version="$(quiet_safe_cd "$HOMEBREW_PREFIX/bin" && pwd -P)/$SCM_FILE"
```

Comment:
It's set as soon as Homebrew starts up, so it should be available in all Homebrew subprocesses, and this script should never be invoked outside of Homebrew, right?